openwisp_utils/admin_theme/static/lib/netjsongraph/css/netjsongraph.css

```
@@ -0,0 +1,382 @@
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  padding: 0 20px;
+  margin-top: 20px;
+}
+
+.njg-container .njg-keyLabel {
+  font-size: 14px;
+  font-weight: 600;
+  flex: 1 1 50%;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)